Week2/homework/Exercise2.js

```diff
@@ -0,0 +1,14 @@
+//Exercise1
+
+'use strict';
+
+for (let i = 0; i <= 20; i++){
+    if(i % 2 === 1){
+        console.log(`The number ${i} is odd!`)
+    };
+    if(i % 2 === 0){
```

Comment:
Ah good, you could also do this easier with `if else`.

So then it would be:

```
for (let i = 0; i <= 20; i++){
    if(i % 2 === 1){
        console.log(`The number ${i} is odd!`)
    } else {
    console.log(`The number ${i} is even!`)
    }
```

Reply:
Thank you! Now it's revised